src/modules/admin/components/projectConfig/ProjectConfigTable.tsx

```diff
@@ -62,9 +65,24 @@ const columns: ColumnDef<AutoExitConfigFieldsFragment>[] = [
         <NotCollectedText variant='body2'>Any Organization</NotCollectedText>
       ),
   },
+  {
+    header: 'Config Options',
+    render: ({ configOptions }) => {
+      if (!configOptions) return;
+      const parsedOptions = JSON.parse(configOptions);
+
+      return Object.keys(parsedOptions).map((key) => {
+        return (
+          <span key={key}>
+            {capitalize(key.replaceAll(/_/g, ' '))}: {parsedOptions[key]}
+          </span>
+        );
+      });
+    },
+  },
 ];
 
-const AutoExitTable = ({
+const ProjectConfigTable = ({
   setSelectedId,
 }: {
   selectedId?: string | null;
```

Comment:
I know this is legacy but it doesn't look like `selectedId` is used at all. This is more of a "click" than a selection action. I would suggested renaming the `setSelectedId` prop to  `onClickRow`, which is a bit more descriptive of what it's actually doing. The caller would do `onClickRow={setSelected}`.